roles/docker/tasks/main.yml

```diff
@@ -67,7 +67,7 @@
     dest: /etc/sysconfig/docker
     regexp: '^{{ item.reg_conf_var }}=.*$'
     line: "{{ item.reg_conf_var }}='{{ item.reg_fact_val | oo_prepend_strings_in_list(item.reg_flag ~ ' ') | join(' ') }}'"
-  when: item.reg_fact_val != '' and  docker_check.stat.isreg
+  when: item.reg_fact_val != '' and  (docker_check.stat.isreg is define and docker_check.stat.isreg)
```

Comment:
@abutcher I followed just the example from [ansible stat documentation](http://docs.ansible.com/ansible/stat_module.html).
If you follow the docker install instructions from [docker site](https://docs.docker.com/engine/installation/linux/rhel/) you will hit this issue